src/ship.js

```diff
@@ -0,0 +1,25 @@
+class Ship {
+    constructor(itinerary) {
+        this.itinerary = itinerary;
+        this.itineraryCount = 0;
+        this.currentPort = itinerary.ports[0];
+        this.previousPort = null;
+        this.currentPort.addShip(this);
+    }
+    setSail() {
```

Comment:
Common convention is to leave a clear line between each of your methods to make it easier to read, should do this on lines 19-20 too:
```
    }

    setSail() {
```